sponsors.html

```diff
@@ -62,7 +62,49 @@
 
 		  <!-- Three columns of text below the carousel -->
 		  <div class="row">
-				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code</p>
+		  		<br />
+		  		<br />
+				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code. 
+				<br />
+				Interested in donating resources to a club? Click <a href="#" data-toggle="modal" data-target="#sponsorModal">here</a>
+
+					<div class="modal fade sponsorModal" tabindex="-1" role="dialog" aria-labelledby="sponsorModal" aria-hidden="true" id="bannerformmodal">
+						<div class="modal-dialog modal-sm">
+        						<div class="modal-content">
+          						<div class="modal-header">
+	                					<button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+	                					<h4 class="modal-title">Sponsor Interest Form</h4>
+                					</div>
+                					<div class="modal-body">
+                     						<div id="wufoo-mobbd7d0s4mjkm">
+									Fill out my <a href="https://interfaceschool.wufoo.com/forms/mobbd7d0s4mjkm">online form</a>.
+								</div>
+									<script type="text/javascript">var mobbd7d0s4mjkm;(function(d, t) {
+									var s = d.createElement(t), options = {
+									'userName':'interfaceschool',
+									'formHash':'mobbd7d0s4mjkm',
+									'autoResize':true,
+									'height':'832',
+									'async':true,
+									'host':'wufoo.com',
+									'header':'show',
+									'ssl':true};
+									s.src = ('https:' == d.location.protocol ? 'https://' : 'http://') + 'www.wufoo.com/scripts/embed/form.js';
+									s.onload = s.onreadystatechange = function() {
+									var rs = this.readyState; if (rs) if (rs != 'complete') if (rs != 'loaded') return;
+									try { mobbd7d0s4mjkm = new WufooForm();mobbd7d0s4mjkm.initialize(options);mobbd7d0s4mjkm.display(); } catch (e) {}};
+									var scr = d.getElementsByTagName(t)[0], par = scr.parentNode; par.insertBefore(s, scr);
+									})(document, 'script');
+								</script>
+								</div>
+              						<div class="modal-footer">
+                						<button type="button" class="btn btn-blue">Submit</button>
+              						</div>          
+        						</div>
+        						</div>
+      						</div>
+    					</div>
+				to complete a short form and our fundraising coordinator will contact you for more details.</p>
```

Comment:
Probably put the modal div element somewhere besides between the Click here and ...to complete a short form. It looks broken up in my browser.
<img width="536" alt="screen shot 2016-07-10 at 7 41 04 pm" src="https://cloud.githubusercontent.com/assets/2250413/16717170/5b6d8494-46d6-11e6-834b-0f254e561ac0.png">